pkg/controller/migmigration/task.go

```diff
@@ -39,52 +39,82 @@ const (
 	Completed                     = "Completed"
 )
 
-// Steps
-var Step = []string{
-	Created,
-	Started,
-	Prepare,
-	EnsureInitialBackup,
-	InitialBackupCreated,
-	InitialBackupFailed,
-	AnnotateResources,
-	EnsureStagePods,
-	StagePodsCreated,
-	RestartRestic,
-	ResticRestarted,
-	QuiesceApplications,
-	EnsureQuiesced,
-	EnsureStageBackup,
-	StageBackupCreated,
-	StageBackupFailed,
-	EnsureStageBackupReplicated,
-	EnsureStageRestore,
-	StageRestoreCreated,
-	StageRestoreFailed,
-	EnsureStagePodsDeleted,
-	EnsureAnnotationsDeleted,
-	EnsureInitialBackupReplicated,
-	EnsureFinalRestore,
-	FinalRestoreCreated,
-	FinalRestoreFailed,
-	Completed,
+// Flags
+const (
+	QUIESCE = 0x01 // Only when QuiescePods (true).
+	PVS     = 0x02 // Only when PVs migrated.
+)
+
+type Itinerary []Step
+
+var StageItinerary = Itinerary{
+	{phase: Created},
+	{phase: Started},
+	{phase: Prepare},
+	{phase: AnnotateResources},
```

Comment:
Prior logic would have this with `flags: PVS` as well.

Reply:
Fixed.